### ISSUE TITLE
pac modelbuilder - support javascript language

### ISSUE THREAD

**Issue by kalluu-ck** [microsoft/PowerPlatform-DataverseModelBuilder#5]
Dear team,

In my project, we work a lot with Web API, and now we don't want to hardcode the logical / schema names of entity fields.
So is there a way of using the pac modelbuilder --language=javascript to generate js files with all the needed metadata (schema names, logical names of entity, fields and relationships) of an entity? Then we can just import and use them instead of hardcoding.

Thanks,
Kim Anh

**Comment by daryllabar** [comment]
There are a couple options available, but not in the model builder. https://discord.com/channels/788761488188899329/804533342653120532/1131373640856916028

**Comment by MattB-msft** [comment]
This is something we would like to provide support for, targeting Microsoft.System.Text.JSON's and NewtonSofts converters,  but have not yet had time to get to it. 

As @daryllabar  said, there are a few solutions available on from the community for this at this time.

**Comment by kalluu-ck** [comment]
Yeah, I know there're some community tools for that, however, I love using pac for all tasks related to Power Apps :)
I'm looking forward to seeing the new feature comes true.
